docs/docs/examples-tutorials/recipes/running-tracetest-with-aws-terraform.md

```diff
@@ -0,0 +1,266 @@
+
+[Terraform](https://www.terraform.io/) is an infrastructure as code tool that lets you define both cloud and on-prem resources in human-readable configuration files that you can version, reuse, and share. You can then use a consistent workflow to provision and manage all of your infrastructure throughout its lifecycle.
+
+[AWS Fargate](https://aws.amazon.com/fargate/) is a serverless, pay-as-you-go compute engine that lets you focus on building applications without managing servers. AWS Fargate is compatible with both Amazon Elastic Container Service (ECS) and Amazon Elastic Kubernetes Service (EKS).
+
+## Sample Node.js Serverless API with Jaeger, OpenTelemetry, AWS Fargate and Tracetest
+
+This is a simple quick start on how to deploy a Node.js Serverless API to use OpenTelemetry instrumentation with traces and Tracetest for enhancing your e2e and integration tests with trace-based testing. The infrastructure will use Jaeger as the trace data store, and OpenTelemetry Collector to receive traces from the Node.js app and Terraform to provision the required AWS services to run Tracetest in the cloud.
+
+## Services Architecture
+
+![Terraform Serverless Diagram](../img/terraform-serverless-diagram.png)
+
+## Prerequisites
+
+You will need [Terraform](https://www.terraform.io/), a configured instance of the [AWS CLI](https://aws.amazon.com/cli/) installed on your machine to run this quick start Serverless API!
+
+## Project Structure
+
+The project is built with Terraform. It contains multiple `.tf` files where each part of the infra is divided.
+
+### 1. Node.js Serverless API
+
+The `src` folder is the root directory that contains the Node.js files for both the API and the OpenTelemetry instrumentation.
+The infrastructure required to run the serverless API can be find as part of the `api.tf` file.
+
+### 2. Tracetest
+
+The `tracetest.tf` file contains the different services and dependencies to run the Tracetest server using AWS Fargate.
+
+### 3. Jaeger
+
+Inside the `jaeger.tf` file you'll find the required services to run the all in one instance using AWS Fargate.
+
+### AWS Network
+
+To connect all of the services, the example generates a VPC network which includes private and public subnets. Internal service like the Postgres RDS instance are protected behind the VPC and only accessible through a node within the internal network.
+
+There are two [Application Load Balancers](https://aws.amazon.com/elasticloadbalancing/application-load-balancer/), a public balancer which provides access to the Tracetest task instance through port `11633` and to the Jaeger UI from `16686`.
+
+The second is an internal load balancer that allows the instrumented Node.js Lambda functions connect to the Jaeger OTLP endpoint using a dns instead of an IP.
```

Comment:
```suggestion
The second is an internal load balancer that allows the instrumented Node.js Lambda functions connect to the Jaeger OTLP endpoint using a DNS instead of an IP.
```